# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../acl/acl-add-destdir.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./include/buildmacros.orig	2012-12-30 19:52:35.262553946 +0100
+++ ./include/buildmacros	2012-12-30 19:56:34.532084427 +0100
@@ -61,16 +61,16 @@
 LTCOMPILE = $(LIBTOOL) --mode=compile $(CCF)
 
 ifeq ($(ENABLE_SHARED),yes)
-LTLDFLAGS += -rpath $(PKG_LIB_DIR)
+LTLDFLAGS += -rpath $(DESTDIR)$(PKG_LIB_DIR)
 LTLDFLAGS += -version-info $(LTVERSION)
 endif
 
 ifeq ($(ENABLE_SHARED),yes)
 INSTALL_LTLIB = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_LIB_DIR); \
-	../$(INSTALL) -m 644 -T so_dot_version $(LIBNAME).lai $(PKG_LIB_DIR); \
-	../$(INSTALL) -T so_dot_current $(LIBNAME).lai $(PKG_LIB_DIR)
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -m 644 -T so_dot_version $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -T so_dot_current $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR)
 endif
 
 # Libtool thinks the static and shared libs should be in the same dir, so
@@ -82,15 +82,15 @@
 ifeq ($(ENABLE_SHARED),yes)
 INSTALL_LTLIB_DEV = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 $(LIBNAME).lai $(PKG_DEVLIB_DIR)/$(LIBNAME).la ; \
-	../$(INSTALL) -m 755 -d $(PKG_LIB_DIR); \
-	../$(INSTALL) -T so_base $(LIBNAME).lai $(PKG_LIB_DIR); \
-	if test "x$(PKG_DEVLIB_DIR)" != "x$(PKG_LIB_DIR)" ; then \
-	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).a $(PKG_LIB_DIR)/$(LIBNAME).a; \
-	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).la $(PKG_LIB_DIR)/$(LIBNAME).la; \
-	../$(INSTALL) -S $(PKG_LIB_DIR)/$(LIBNAME).so $(PKG_DEVLIB_DIR)/$(LIBNAME).so; \
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR)/$(LIBNAME).la ; \
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -T so_base $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR); \
+	if test "x$(DESTDIR)$(PKG_DEVLIB_DIR)" != "x$(DESTDIR)$(PKG_LIB_DIR)" ; then \
+	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).a $(DESTDIR)$(PKG_LIB_DIR)/$(LIBNAME).a; \
+	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).la $(DESTDIR)$(PKG_LIB_DIR)/$(LIBNAME).la; \
+	../$(INSTALL) -S $(PKG_LIB_DIR)/$(LIBNAME).so $(DESTDIR)$(PKG_DEVLIB_DIR)/$(LIBNAME).so; \
 	fi
 else
 INSTALL_LTLIB_DEV = $(INSTALL_LTLIB_STATIC)
@@ -98,8 +98,8 @@
 
 INSTALL_LTLIB_STATIC = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR)
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR)
 
 INSTALL_MAN = \
 	@for d in $(MAN_PAGES); do \
@@ -111,7 +111,7 @@
 				-e 's/\\\f[0-9]//g' -e 's/  / /g;q'`; \
 		do \
 			[ -z "$$m" -o "$$m" = "\\" ] && continue; \
-			t=$(MAN_DEST)/$$m.$(MAN_SECTION); \
+			t=$(DESTDIR)$(MAN_DEST)/$$m.$(MAN_SECTION); \
 			if $$first; then \
 				if $(HAVE_ZIPPED_MANPAGES); then \
 					$(ZIP) -9 -c $$d > $$d.gz; _sfx=.gz; \
@@ -131,7 +131,7 @@
 INSTALL_LINGUAS = \
 	@for l in $(LINGUAS) ""; do \
 		if test -f "$$l.mo" ; then \
-			ldir=$(PKG_LOCALE_DIR)/$$l/LC_MESSAGES; \
+			ldir=$(DESTDIR)$(PKG_LOCALE_DIR)/$$l/LC_MESSAGES; \
 			$(INSTALL) -m 755 -d $$ldir; \
 			$(INSTALL) -m 644 $$l.mo $$ldir/$(PKG_NAME).mo; \
 		fi; \
--- ./Makefile.orig	2012-12-30 19:58:10.218286182 +0100
+++ ./Makefile	2012-12-30 19:58:42.183036270 +0100
@@ -89,8 +89,8 @@
 	fi
 
 install: default $(addsuffix -install,$(SUBDIRS))
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 README $(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 README $(DESTDIR)$(PKG_DOC_DIR)
 
 install-dev: default $(addsuffix -install-dev,$(SUBDIRS))
 
--- ./chacl/Makefile.orig	2012-12-30 20:00:09.965052538 +0100
+++ ./chacl/Makefile	2012-12-30 20:00:33.905608325 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
 install-dev install-lib:
--- ./getfacl/Makefile.orig	2012-12-30 20:01:07.586372402 +0100
+++ ./getfacl/Makefile	2012-12-30 20:01:30.414929275 +0100
@@ -31,6 +31,6 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
 install-dev install-lib:
--- ./setfacl/Makefile.orig	2012-12-30 20:03:21.681503026 +0100
+++ ./setfacl/Makefile	2012-12-30 20:03:50.574164225 +0100
@@ -31,6 +31,6 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
 install-dev install-lib:
--- ./man/man1/Makefile.orig	2012-12-30 20:05:05.547903528 +0100
+++ ./man/man1/Makefile	2012-12-30 20:06:05.113263619 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install : default
-	$(INSTALL) -m 755 -d $(MAN_DEST)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(MAN_DEST)
 	$(INSTALL_MAN)
 install-dev install-lib:
--- ./man/man3/Makefile.orig	2012-12-30 20:05:11.448043609 +0100
+++ ./man/man3/Makefile	2012-12-30 20:05:49.820907660 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install-dev : default
-	$(INSTALL) -m 755 -d $(MAN_DEST)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(MAN_DEST)
 	$(INSTALL_MAN)
 install install-lib:
--- ./man/man5/Makefile.orig	2012-12-30 20:05:15.880144314 +0100
+++ ./man/man5/Makefile	2012-12-30 20:05:35.016561275 +0100
@@ -30,6 +30,6 @@
 include $(BUILDRULES)
 
 install : default
-	$(INSTALL) -m 755 -d $(MAN_DEST)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(MAN_DEST)
 	$(INSTALL_MAN)
 install-dev install-lib:
--- ./doc/Makefile.orig	2012-12-30 20:07:45.935615652 +0100
+++ ./doc/Makefile	2012-12-30 20:08:39.948873200 +0100
@@ -31,12 +31,12 @@
 	$(ZIP) --best -c < CHANGES > $@
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 PORTING CHANGES.gz $(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 PORTING CHANGES.gz $(DESTDIR)$(PKG_DOC_DIR)
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -S CHANGES.gz $(PKG_DOC_DIR)/changelog.gz
+	$(INSTALL) -S CHANGES.gz $(DESTDIR)$(PKG_DOC_DIR)/changelog.gz
 else
-	$(INSTALL) -m 644 COPYING COPYING.LGPL $(PKG_DOC_DIR)
+	$(INSTALL) -m 644 COPYING COPYING.LGPL $(DESTDIR)$(PKG_DOC_DIR)
 endif
 
 install-dev install-lib:
--- ./include/Makefile.orig	2012-12-30 20:09:09.129564720 +0100
+++ ./include/Makefile	2012-12-30 20:09:56.070611199 +0100
@@ -31,10 +31,10 @@
 include $(BUILDRULES)
 
 install-dev: default
-	$(INSTALL) -m 755 -d $(PKG_INC_DIR)
-	$(INSTALL) -m 755 -d $(PKG_INC_DIR)/sys
-	$(INSTALL) -m 644 acl.h $(PKG_INC_DIR)/sys/acl.h
-	$(INSTALL) -m 755 -d $(PKG_INC_DIR)/acl
-	$(INSTALL) -m 644 libacl.h $(PKG_INC_DIR)/acl/libacl.h
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_INC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_INC_DIR)/sys
+	$(INSTALL) -m 644 acl.h $(DESTDIR)$(PKG_INC_DIR)/sys/acl.h
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_INC_DIR)/acl
+	$(INSTALL) -m 644 libacl.h $(DESTDIR)$(PKG_INC_DIR)/acl/libacl.h
 
 install install-lib:
